api/v1alpha4/condition_consts.go

```diff
@@ -201,3 +201,14 @@ const (
 	// from making any further remediations.
 	TooManyUnhealthyReason = "TooManyUnhealthy"
 )
+
+// Conditions and condition Reasons for  MachineDeployments
+
+const (
+	// AvailableCondition means the MachineDeployment is available, ie. at least the minimum available
+	// machines required are up and running for at least minReadySeconds.
+	AvailableCondition ConditionType = "Available"
+
+	// MinimumMachinesAvailableReason reflects the minimum availability of machines for a machinedeployment.
```

Comment:
I'm having a hard time to read this, "reflects the minimum availability of machines for a machinedeployment." doesn't seem right to me, since this variable is just meant to be the reason for AvailableCondition=false. Also shouldn't it be `NotMinimumMachinesAvailable`?

Reply:
Changed it to `NotMinimumMachinesAvailableReason` and updated the comment.